packages/renderer/src/lib/ui/Badge.spec.ts

```diff
@@ -89,3 +89,13 @@ test('Should display badge with dark color', async () => {
   // check color being there in the style
   expect(label).toHaveStyle('background-color: rgb(0, 255, 0)');
 });
+
+test('Should display badge with custom class', async () => {
+  render(Badge, { label: 'customLabel', class: 'text-right' });
+
+  // expect to see label inside the label
+  const label = screen.getByText('customLabel');
+  expect(label).toBeInTheDocument();
+
+  vi.waitFor(() => label.classList.contains('text-right'));
```

Comment:
```suggestion
  vi.waitFor(() => expect(label).toHaveClass('text-right'));
```

Reply:
Shouldn't you be waiting for the vi.waitFor ? it is not blocking

Reply:
yes it was testing nothing

should be fixed now 👍